backend/packages/Upgrade/.env.docker.local.example

```diff
@@ -77,11 +77,10 @@ EMAIL_BUCKET="s3_bucket"
 #
```

Comment:
is this change just to prefer having a real boolean in there, yeah? that's fine, just want to make sure you meant to commit this.

Reply:
Yes, I meant to commit this.